src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -865,10 +872,13 @@ void ClusterResourceScheduler::UpdateLocalAvailableResourcesFromResourceInstance
   auto local_view = it_local_node->second.GetMutableLocalView();
   for (size_t i = 0; i < PredefinedResources_MAX; i++) {
     local_view->predefined_resources[i].available = 0;
+    local_view->predefined_resources[i].total = 0;
```

Comment:
Incidental bug fix that the unit test hits (actually i wonder if this could be related to the placement group instability issues?)

Reply:
i think it could be. the value should be undefined.

Reply:
Is there a separate unit test that we can write for this that we can split out from this PR?